src/Util.ts

```diff
@@ -118,25 +118,28 @@ export function objectFrom(value: any, params: { withType: boolean; noEmptyStrin
 }
 
 /**
- * Returns entries from something iterable, such as a Map, Set, or Array
- * If value is null it returns an empty array
- * @param value iterable input
- * @returns An javascript object
+ * Returns entries from an iterable input like Map, Set, or Array.
+ *
+ * For all other types of values (including `null` or `undefined`), it returns an empty array.
+ *
+ * @param value An iterable input
+ * @returns An array of key-value pairs
  */
-export function objectEntries(value: any | null): [string, any][] {
+export function objectEntries(value: any): [string, any][] {
```

Comment:
Sorry for asking too much, but can we do something like this, to have a strict typing better / better intellisense? or will it break some stuff in your repos?

```suggestion
export function objectEntries<T extends Record<string, any>>(value: T): [keyof T, T[keyof T]][] {
```